src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/ReceivedMessagesHealthEndpoint.java

```diff
@@ -28,12 +35,13 @@ public class ReceivedMessagesHealthEndpoint implements HealthIndicator {
     @Autowired
     private HolidayService holidayService;
 
-    @Override
-    public Health getHealth(boolean includeDetails) {
-        return HealthIndicator.super.getHealth(includeDetails);
-    }
-
-    @Override
+    @Operation(description = "Query case event message db to find problematic messages by job name")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "OK", content = {
+            @Content(mediaType = "application/json", schema = @Schema(implementation = Health.class))})
+    })
+    @GetMapping(path = "/ccdMessagesReceived/health")
```

Comment:
I would probably name this path differently

Reply:
👍